integration/__tests__/matic_scen.js

```diff
@@ -1,31 +1,120 @@
+        { token: 'comp' }
     ],
     validators: ['alice', 'bob'],
-    actors: ['ashley', 'darlene'],
+    actors: ['ashley', 'bert', 'chuck', 'darlene', 'edward'],
     chain_opts: {
         matic: {
             name: 'matic',
-            provider: 'ganache', // [env=PROVIDER]
+            provider: 'ganache',
             ganache: {
                 opts: {},
                 web3_port: null
             },
         },
     },
+    native: true, // for log messages - removes wasm:stripped - no need for wasm no forkless upgrade testing here
+    freeze_time: now,
+    initial_yield: 300,
+    initial_yield_start_ms: now
 };
 
 buildScenarios('Matic', lock_scen_info, [
     {
-        skip: true, // TODO FIX SCEN
-        name: 'Matic',
-        scenario: async ({ darlene, maticZrx }) => {
-            await darlene.lock(1000, maticZrx);
+        name: 'Matic Lock',
+        scenario: async ({ashley, usdc, darlene, maticZrx }) => {
+            // Lock on Polygon
+            let balance, liquidity;
+            await darlene.lock(100, maticZrx);
+            balance = await darlene.balanceForToken(maticZrx);
+            let darleneLiquidity = await darlene.liquidity();
+            expect(balance).toEqual(100);
+            expect(darleneLiquidity).toBeGreaterThan(0);
+
+            // lock on ethereum
+            await ashley.lock(1000, usdc);
+            balance = await ashley.balanceForToken(usdc);
+            liquidity = await ashley.liquidity();
+            expect(balance).toEqual(1000);
+            expect(liquidity).toBeGreaterThan(0);
+
+            // lock on polygon again
+            await darlene.lock(100, maticZrx);
+            balance = await darlene.balanceForToken(maticZrx);
+            liquidity = await darlene.liquidity();
+            expect(balance).toEqual(200); // double the asset balance (no interest, 0 utilization)
+            expect(liquidity).toEqual(2*darleneLiquidity); // liquidity must have gone up
+        }
+    },
+    {
+        name: 'Collateral Borrowed Interest Lump Sum',
+        scenario: async ({ darlene, chuck, cash, chain, usdc, maticZrx}) => {
+            // await prices.postPrices();
+            let balance, liquidity;
+            await chain.setFixedRate(usdc, 500); // 5% APY fixed
+            await darlene.lock(10000, maticZrx);
+            balance = await darlene.balanceForToken(maticZrx);
+            let liquidity1 = await darlene.liquidity();
+            expect(balance).toEqual(10000);
+            expect(liquidity1).toBeGreaterThan(0);
+            // now we know everything is in order, make the transfer, let's say half our liquidity worth
+            await darlene.transfer(1000, usdc, chuck);
+            expect(await darlene.chainBalance(usdc)).toEqual(-1000);
+            expect(await chuck.chainBalance(usdc)).toEqual(1000);
+            // xxx todo:wn this test is broken but so is its friend, interest is accruing even though time is not "on"
```

Comment:
Yeah I can work on this test

Reply:
TL;DR - it turns out that the problem is that I was building without the `integration` feature. 

I was doing this because I generally preferred to use debug profile. I mostly have this preference because as I work on rust code I often `cargo build` when i'm ready. If I then `cargo build --release` as in `yarn build` it ends up having to build everything again from scratch. 

Compile times are pretty painful which lead me down this road generally. Cargo check isn't too much better because ultimately you're going to want a build and when you're ready for a build it needs to do the process from scratch anyway. At least rust has incremental builds so that when you don't touch a lot of the code it doesn't have to rebuild much.